index.js

```diff
@@ -12,7 +12,8 @@ try {
  * Element prototype.
  */
 
-var proto = Element.prototype;
+var Element = require('global-object').Element;
```

Comment:
Do you need to add another dep?

It appears to me that just...
```js
 var proto = typeof Element !== 'undefined' ? Element.prototype : {};
```

...would achieve what is desired.

I understand that `global-object` is tiny and its entire purpose is as you're using it, but a more surgical PR might be more welcome.